packages/frontend-core/src/components/FilterBuilder.svelte

```diff
@@ -124,6 +124,7 @@
     const fieldSchema = schemaFields.find(x => x.name === filter.field)
     filter.type = fieldSchema?.type
     filter.subtype = fieldSchema?.subtype
+    filter.formulaType = fieldSchema?.formulaType
```

Comment:
The formula type was missing from the structure we pass into the lucene utils in shared core, which resulted in us being unable to detect static formulae.